projects.go

```diff
@@ -157,6 +157,7 @@ type Project struct {
 	MergePipelinesEnabled                    bool               `json:"merge_pipelines_enabled"`
 	MergeTrainsEnabled                       bool               `json:"merge_trains_enabled"`
 	RestrictUserDefinedVariables             bool               `json:"restrict_user_defined_variables"`
+	CIPipelineVariablesMinimumOverrideRole   AccessControlValue `json:"ci_pipeline_variables_minimum_override_role"`
```

Comment:
I chose `AccessControlValue` to match existing code, but  for what it's worth, the `AccessControlValue` type seems to have been originally intended for [something completely different](https://github.com/vindvaki/go-gitlab/blob/c4c9b8a1e0290654872f642e3b9f04f084d939ac/types.go#L40-L50). Over time, it seems to have morphed into more of a catch-all string type for different access control values 🤷

Reply:
@vindvaki  - I agree, I'm not sure `AccessControlValue` is the right type here. Since the API seems like it's accepting a string value for the Access Level instead of the integer value (which is usually defined using the `AccessLevelValue` type), I think we should probably create a new type for it in `types.go`, especially since `no_one_allowed` seems to deviate from the normal `no_one` that's used elsewhere.

would you mind doing that? Otherwise this LGTM!

Reply:
@RicePatrick sure, I've created a new type in https://github.com/xanzy/go-gitlab/pull/2057/commits/fc7656ee9b99ff91c8a174ef2456576d0ad4d420. Let me know what you think 🏓

Reply:
@vindvaki  - I apologize for nitpicking, but would you mind adding a comment on the new type to match the values of the others within the file? I tried to push a commit for you, but I don't have permissions to do so :)

I'd suggest something like this:

```golang
// CIPipelineVariablesMinimumOverrideRoleValue represents an access control value
// used for managing access to the CI Pipeline Variable Override feature.
//
// GitLab API docs: https://docs.gitlab.com/ee/api/projects.html
type CIPipelineVariablesMinimumOverrideRoleValue = string

// List of available CIPipelineVariablesMinimumOverrideRoleValue levels.
//
// GitLab API docs: https://docs.gitlab.com/ee/api/projects.html
const (
	CIPipelineVariables_NoOneAllowedRole CIPipelineVariablesMinimumOverrideRoleValue = "no_one_allowed"
	CiPipelineVariables_OwnerRole        CIPipelineVariablesMinimumOverrideRoleValue = "owner"
	CiPipelineVariables_MaintainerRole   CIPipelineVariablesMinimumOverrideRoleValue = "maintainer"
	CIPipelineVariables_DeveloperRole    CIPipelineVariablesMinimumOverrideRoleValue = "developer"
)

```

Reply:
@RicePatrick thanks, done 🏓